include/vinecopulib/vinecop/implementation/tools_select.ipp

```diff
@@ -182,23 +182,7 @@ VinecopSelector::sparse_select_all_trees(const Eigen::MatrixXd& data)
   double d = static_cast<double>(d_);
 
   std::vector<double> thresholded_crits;
-  if (controls_.get_select_threshold()) {
```

Comment:
Same question: why do you remove that?

Reply:
I initially thought it was smart to warm start the threshold search with a value calculated form the unconditional Kendall's tau. But the problem is that the very first model that we try might already be worse than an independence model. In that case, the algorithm stopped without having any model at hand, which caused a segfault. Now we simply start with threshold = 1 (which fits an independence model) and then start a proper search.